doc/scenarious.md

```diff
@@ -0,0 +1,126 @@
+
+# Handling Multiple Tables in the SQL Query Builder
```

Comment:
As you name your document as it should describe something, as for me this document is not about handling multiply tables, but rather about scenarios that should be supported

As mentioned few lines below